assignment/api/src/java/org/sakaiproject/assignment/api/persistence/AssignmentRepository.java

```diff
@@ -91,4 +92,15 @@ AssignmentSubmission newSubmission(String assignmentId,
     String findAssignmentIdForGradebookLink(String context, String linkId);
 
     Collection<String> findGroupsForAssignmentById(String assignmentId);
+    
+    AssignmentSubmissionSubmitter findSubmissionSubmitter(String submitterId);
```

Comment:
This will likely return a list not a single when searching for a users submissions

Reply:
This method isn't used anymore.  Deleted.